share/locale/mscore_fr.ts

```diff
@@ -17599,7 +17600,7 @@ veuillez choisir un nom différent :</translation>
     <message>
         <location filename="../../mscore/pagesettings.ui" line="220"/>
         <source>Inch (in)</source>
-        <translation>Pouce (p)</translation>
+        <translation>Inch (in)</translation>
```

Comment:
Why? In German I translated to to 'Zoll (")` (and the ìn` above too to `"`

Reply:
@Jojo-Schmitz
Because in France the 2 words are accepted 'Pouce' is the translation of 'Inch' but the 'p' in faulty, it is a measure and should not be translated, as 'mm' 'kg' etc.
Therefore, I can translate as 'Pouce (in)'
Unfortunately, I see I have not respected the French grammatical rules !!! As I have do in the 4.x translations.
Correct translations should be 'Inches (in)' and 'Millimètres (mm)'
The included file have this correction. plus the correction of the 'fr_FR' by 'fr' in the .ts file.
Let me know if this is ok.
[mscore_fr.ts.zip](https://github.com/user-attachments/files/19522977/mscore_fr.ts.zip)

Reply:
It has the same issue with fr vs fr_FR ;-)

Reply:
Other than that it is OK and I'll take it

Reply:
@Jojo-Schmitz 
Thanks to have merged this.
Could it be possible that the config of QT Linguist do the change of 'fr' in 'fr_FR' ? I am using 6.2.4.
I will do some tests to understand.
Have a good day.

Reply:
It doesn't do this for me, it does not turn de into de_DE

Reply:
@Jojo-Schmitz 
Problem found:
- Each time I open a .ts file the config choose the main country of a language

![QtLinguist-Config-FR-A1](https://github.com/user-attachments/assets/d60bde87-d3ca-44fc-815d-697491056853)

- The drop down list all countries that share the lanquage
- I have to manually change to 'All countries (Tous le pays)' before saving to have only the country code 'fr'

![QtLinguist-Config-FR-B1](https://github.com/user-attachments/assets/b5b0696c-3e67-4ac1-8545-6d7802df74e3)

- Trying with your 'mscore_de.ts' your main country is selected

![QtLinguist-Config-DE](https://github.com/user-attachments/assets/ab536427-3d7c-467d-97da-2b2021dc91a2)

Not yet find any config file nor registry entry for Linguist...

Reply:
I'm using Qt Linguist 5.15.2
Opening mscore_fr.ts and checking settings gives:
![image](https://github.com/user-attachments/assets/d404a5ed-ec5a-4b39-a01e-3a8d23bcc69f)
For German it gives
![image](https://github.com/user-attachments/assets/2ebb7347-41ee-42d6-912b-631f6e074388)

However, when using Qt Linguist 6.2.4 it shows:
![image](https://github.com/user-attachments/assets/7b178be5-747b-498f-a7b6-9e2398ec63f8)

So it is those versions that add it. As 3.7 is cimpiled with 5.15.2, I've been use that...

Reply:
OK, I will reinstall 5.15.2 to avoid anothers hidden bugs or surprises, I have not see errors when using my file in 3.7, but who knows...